core/utils/arp.h

```diff
@@ -0,0 +1,42 @@
+#ifndef BESS_UTILS_ARP_H_
+#define BESS_UTILS_ARP_H_
+
+#include "endian.h"
+#include "ether.h"
+#include "ip.h"
+
+namespace bess {
+namespace utils {
+
+// A basic ARP header definition.
+struct[[gnu::packed]] ARP {
```

Comment:
Google style guide says it should be `Arp` (just like `Tcp`).

... Sorry I hate the naming scheme too :(